src/status_im/data_store/realm/schemas/account/v8/transport.cljs

```diff
@@ -0,0 +1,38 @@
+(ns status-im.data-store.realm.schemas.account.v8.transport
+  (:require
+   [clojure.string :as string]
+   [taoensso.timbre :as log]))
+
+(def schema {:name       :transport
+             :primaryKey :chat-id
+             :properties {:chat-id               :string
+                          :ack                   :string
+                          :seen                  :string
+                          :pending-ack           :string
+                          :pending-send          :string
+                          :topic                 :string
+                          :one-to-one            {:type    :bool
+                                                  :optional true}
+                          :fetch-history?        {:type    :bool
+                                                  :default false}
+                          :resend?               {:type     :string
+                                                  :optional true}
+                          :sym-key-id            {:type    :string
+                                                  :optional true}
+                          ;;TODO (yenda) remove once go implements persistence
+                          :sym-key               {:type     :string
+                                                  :optional true}}})
+
+(defn one-to-one? [chat-id]
+  (re-matches #"^0x[0-9a-fA-F]+$" chat-id))
+
+(defn migration [old-realm new-realm]
+  (log/debug "migrating transport chats")
+  (let [old-chats (.objects old-realm "transport")
+        new-chats (.objects new-realm "transport")]
+    (dotimes [i (.-length old-chats)]
+      (let [old-chat (aget old-chats i)
+            new-chat (aget new-chats i)
+            chat-id  (aget old-chat "chat-id")]
+        (when (one-to-one? chat-id)
+          (aset new-chat "one-to-one" true))))))
```

Comment:
Just an idea, but wouldn't be better to just fetch the information if the chat is 1-1 or not by looking up to `chat` object (realm or in-memory) based on the link that `:chat-id` is the same for transport and chat objects ?
We already hold the type-of-chat information for `:chat` object so we would be spared any data duplication + migration would not be necessary...

Reply:
Yes, I agree that probably would be the simplest way,
at the moment though we don't always create a chat object, as we don't have a `show?` setting (so if it's there we always display it), so that is the only way unless we change that logic, which probably is best to do once we decide to allow the user to `hide` chats in the mainview.